gsoc_contributors_guide.md

```diff
@@ -0,0 +1,33 @@
+# GSoC Contributor's Guide
+
+The Echoshader team aims to recruit talented [Google Summer of Code (GSoC)](https://summerofcode.withgoogle.com/) participants to help us create the capability to interactively visualize large volumes of cloud-hosted ocean sonar data to accelerate the the data exploration and discovery process. This project will go hand-in-hand with ongoing development of the [echopype](https://github.com/OSOceanAcoustics/echopype) package that handles the standardization, pre-processing, and organization of these data. We aim for creating ocean sonar data visualization functionalities based on the [HoloViz](https://github.com/holoviz/holoviz) suite of tools, and developing widgets that can be flexibly combined in [Panel](https://panel.holoviz.org/) or used in Jupyter notebooks for interactive data visuzliation.
+
+
+## Getting Started
+
+TBD
+
+
+
+
+## Brainstorm with us!
+
+In the [Issues](https://github.com/OSOceanAcoustics/echoshader/issues) section of this repository, we list some visualization ideas from mentors. We encourage you as a GSoC participant to propose your own original ideas as new issues.
+
+Feel free to propose project ideas [raising an issue](https://github.com/OSOceanAcoustics/echoshader/issues/new/choose) in this repo.
+
+After you [sign up as a GSoC participant](https://summerofcode.withgoogle.com/get-started/), feel free to to propose project ideas and ask questions via [raising an issue](https://github.com/OSOceanAcoustics/echoshader/issues/new/choose).
+
+Once the official application opens, please submit your proposals based on the [Echoshader GSoC Proposal](proposal-template.md) template.
+
+
+### Questions?
+
+For project-related question, feel free to [raise an issue] (https://github.com/OSOceanAcoustics/echoshader/issues/new/choose). 
```

Comment:
Can we have questions issue template? So that it can auto label to questions?

Reply:
Sounds good. I'll move the questions part to a new template.